.gitignore

```diff
@@ -10,12 +10,3 @@ npm-debug.log*
 yarn-error.log
```

Comment:
could you please remove it as well?

Reply:
that is part of blueprint so I didn't remove it as it was not showing in the diff